test/media/content_workarounds_integration.js

```diff
@@ -82,39 +82,39 @@ describe('ContentWorkarounds', () => {
     await waiter.waitUntilPlayheadReachesOrFailOnTimeout(video, 5, 30);
   });
 
-  const keySystemsConfigs = new Map()
-      .set('com.widevine.alpha', {
-        servers: {
-          'com.widevine.alpha': 'https://cwip-shaka-proxy.appspot.com/no_auth',
-        },
-      })
-      .set('com.microsoft.playready', {
-        servers: {
-          'com.microsoft.playready': 'http://test.playready.microsoft.com/service/rightsmanager.asmx?cfg=(kid:51745386-2d42-56fd-8bad-4f58422004d7,contentkey:UXRThi1CVv2LrU9YQiAE1w==),(kid:26470f42-96d4-5d04-a9ba-bb442e169800,contentkey:JkcPQpbUXQSpurtELhaYAA==)',
-        },
-      })
-      .set('com.apple.fps', {
-        servers: {
-          'com.apple.fps': 'https://fps.ezdrm.com/api/licenses/b99ed9e5-c641-49d1-bfa8-43692b686ddb',
-        },
-        advanced: {
-          'com.apple.fps': {
-            serverCertificate: null, // empty now, fulfilled during actual test
-          },
+  const keySystemsConfigs = new Map();
+  if (checkWidevineSupport()) {
+    keySystemsConfigs.set('com.widevine.alpha', {
+      servers: {
+        'com.widevine.alpha': 'https://cwip-shaka-proxy.appspot.com/no_auth',
+      },
+    });
+  }
+  if (checkPlayReadySupport()) {
+    keySystemsConfigs.set('com.microsoft.playready', {
+      servers: {
+        'com.microsoft.playready': 'http://test.playready.microsoft.com/service/rightsmanager.asmx?cfg=(kid:51745386-2d42-56fd-8bad-4f58422004d7,contentkey:UXRThi1CVv2LrU9YQiAE1w==),(kid:26470f42-96d4-5d04-a9ba-bb442e169800,contentkey:JkcPQpbUXQSpurtELhaYAA==)',
+      },
+    });
+  }
+  if (checkFairPlaySupport()) {
+    keySystemsConfigs.set('com.apple.fps', {
```

Comment:
I prefer how it has been set up before. Now we will not see pending tests in stats.